Artur_Aghajanyan/Python/25_05_2021/fibonacci.py

```diff
@@ -0,0 +1,24 @@
+fib = input("Give me number for fibonacci: ")
+
+
+if not fib.isdigit():
+    print("Input is string or small than 0")
+else:
+    fib = int(fib)
+    def fibonacci(n):
+        a = 1
+        b = 1
+        if n < 0:
+            print("Incorrect input")
+        elif n == 0:
+            return a
+        elif n == 1:
+            return b
+        else:
+            for i in range(2,n):
+                c = a + b
+                a = b
+                b = c
+            return b
+  
+    print(fibonacci(fib))
```

Comment:
1. ![image](https://user-images.githubusercontent.com/80816144/119568638-c024b880-bdbe-11eb-8fb6-60967c410396.png)
F(0) is incorrect
2. coding style (variable names)
3. could be as recursion

Reply:
new comment